armi/nuclearDataIO/tests/test_xsCollections.py

```diff
@@ -138,6 +138,13 @@ def r(self, r):
         self._r = r
 
     def getVolume(self, *args, **kwargs):
+        """
+        Return the volume of a block.
+
+        .. impl:: Volume of block is retrievable
+            :id: I_ARMI_BLOCK_DIMS10
+            :implements: R_ARMI_BLOCK_DIMS
+        """
```

Comment:
Is this the right spot for the `getVolume` impl tag? I thought the target impl was in blocks.py.

Reply:
Whoops.  Fixed!